cg/cli/backup.py

```diff
@@ -206,7 +210,9 @@ def archive_spring_file(config: CGConfig, spring_file_path: str, dry_run: bool):
 @DRY_RUN
 @click.option("-s", "--sample-id", "object_type", flag_value="sample", type=str)
 @click.option("-c", "--case-id", "object_type", flag_value="case", type=str)
-@click.option("-f", "--flow-cell-id", "object_type", flag_value="flow_cell", type=str)
+@click.option(
+    "-f", "--flow-cell-id", "object_type", flag_value="illumina_run_directory_data", type=str
```

Comment:
I would say limit this PR only to the renaming of the model, and do CLI command changes in another PR